soc/arm/microchip_mec/mec172x/soc.c

```diff
@@ -13,439 +12,11 @@
 #include <arch/cpu.h>
 #include <arch/arm/aarch32/cortex_m/cmsis.h>
 
-#define ECIA_XEC_REG_BASE						\
```

Comment:
Question, does this basically mean no clock selection in this point? 
Would this allow to build and run helloworld with default HW configuration in EVB?

Reply:
Yes, hello world can be compiled but it will not output anything.  This PR is only for compiling the code until enough drivers are updated to do something useful such as UART output.